### DISCUSSION TITLE
Problems updating 2.3.91 to 2.3.100

### DISCUSSION THREAD

**Discussion by HelenH321** [Security-Onion-Solutions/securityonion#7503] [Unsupported Versions]
Hi everyone,

I have a Security Onion Manager instance which I am trying to upgrade from 2.3.91 to 2.3.100, but it keeps hanging on either of the following lines in soup:
"remote: Compressing objects:   0% (1/1621)"
Or 
"Cloning Security Onion github repo into /tmp/sogh/securityonion.
Removing previous upgrade sources.
Cloning the Security Onion Repo.
Cloning into 'securityonion'..."
It gets to either of these points and then just hangs.  I've run it repeatedly and left it varying lengths of time in the hope that it might pick itself back up again, but even after over an hour, no joy.  When looking at the memory resources on the server, they spike up to 90% used at the time I set the script off, but then settle at about 50-60% after.

Would be grateful if anyone has seen this behaviour or knows how I can get around it to update this instance.

Thanks
Helen

**Comment by dougburks** [comment]
Does your machine have access to github.com?

**Reply by HelenH321** [reply]
It does, unfortunately.  I would have loved it to be that.  I successfully updated another instance on a similar subnet last week, and the same changes were made to the proxy both times.

I have been trying again today, with the same behaviour, only today I've been getting varying percentages on the line:
"remote: Compressing objects: 0% (1/1621)" 
So, variously, 34%, 58%, 62% ... etc, and now the (*/*) figure is out of 4798 (~/4798) instead of 1621 as seen last week.

At first these percentages kept going upward every time I kicked off the script, so I thought that perhaps it was still doing the work in the background but not updating the screen, perhaps, and I wasn't being patient enough, but then it reverted to just stalling on 
"Cloning into 'securityonion'..."

Really grateful for any ideas of how to get it moving.  
Thanks
Helen

**Reply by dougburks** [reply]
I don't believe we've had any other reports like this. This really seems like an issue with your proxy or network.
